src/main/java/Exceptions/ArgumentNotFoundException.java

```diff
@@ -0,0 +1,4 @@
+package Exceptions;
+
+public class ArgumentNotFoundException extends Exception{
```

Comment:
Can create specific methods in the exception that print specific error messages, exceptions act just like classes